hermit-sys/build.rs

```diff
@@ -0,0 +1,139 @@
+extern crate walkdir;
+
+use std::env;
+use std::process::Command;
+use walkdir::{DirEntry, WalkDir};
+
+#[cfg(all(not(feature = "rustc-dep-of-std"), not(feature = "with_submodule")))]
+fn build() {
```

Comment:
Just to clarify: We need this distinction, so that the version of hermit-sys pushed to crates.io can reliably download, build and link libos to hermit-sys. Is my understanding correct?

Reply:
Yes, hermit-sys at crates.io is only content of the directors `hermit-sys`. Consequently, a direct access isn't possible, The `hermit-sys` has to download the kernel.

Reply:
Just out of curiosity: I was thinking if it might be possible to add libos as a precompiled library (rlib) to the crates.io package and specifying [an override](https://doc.rust-lang.org/cargo/reference/build-scripts.html#overriding-build-scripts) . This could be done automatically as part of a deploy step and would let us get rid of cargo download. 
In theory the libos rlib should be the same regardless of if it was built on linux, windows or macOS, right? 
I'm not sure if the linking process requires the same compiler version as the one the rlib was built with though.

Reply:
Hm, maybe. I don't know how we can deliver an rlib. Currently, we rebuild everything from scratch.